Sources/SWBUtil/Environment.swift

```diff
@@ -12,25 +12,157 @@
 
 import Foundation
 
-@TaskLocal fileprivate var processEnvironment = ProcessInfo.processInfo.environment
```

Comment:
Ignore the "diffs" in this file. This PR is entirely additions; the original Environment.swift was renamed to EnvironmentHelpers.swift.